src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -124,6 +100,14 @@
       GROUP BY
         strategy)))
 """
+
+STRATEGY_SUBQUERY = """
```

Comment:
Nit: could you rename all of these queries that still require a .format with _FORMAT or _TEMPLATE at the end of their names?